app/models/landing_text.rb

```diff
@@ -0,0 +1,61 @@
+class LandingText < ActiveRecord::Base
+
+  scope :drafts, -> { where(draft: true) }
+  scope :published, -> { where(draft: false) }
+
+  def self.empty
+    LandingText.new(draft: false, locale: nil, texts: empty_texts)
+  end
+
+  def self.current(locale)
+    LandingText.published.where(locale: locale).first
```

Comment:
I would merge with empty_text before returning, so it is automatically completed for `draft` and in controller code. This change will allow easy adding of sections and remove some duplicate `|| Landing.empty` checks.